packages/loader/container-loader/src/container.ts

```diff
@@ -2334,7 +2339,7 @@ export class Container
 	private processRemoteMessage(message: ISequencedDocumentMessage) {
 		// non-interactive clients will not have any pending state we want to save
 		if (this.offlineLoadEnabled && this.isInteractiveClient) {
```

Comment:
this shouldn't need to check, it should just give all ops to the manager, and the manager choose to keep it or not